.travis.yml

```diff
@@ -8,15 +8,15 @@ php:
 
 matrix:
   include:
+    - php: hhvm
```

Comment:
HHVM took 35 minutes to run the tests 0.o
I assume it's not a travis issue, they are slow on wercker too. Probably the slowdown happens for filesystem operations, cli tests run painfully slow (tried them locally and on wercker too)